internal/server/server.go

```diff
@@ -172,3 +174,23 @@ func (s ServerForClient) GetGameState() gamestate.ClientGameState {
 func (s ServerForClient) GetGameConfig() config.ClientConfig {
 	return s.server.gameConfig.GetClientConfig()
 }
+
+func randomUniqueAssignIIGORoles(input []shared.ClientID) (shared.ClientID, shared.ClientID, shared.ClientID) {
```

Comment:
See: https://play.golang.org/p/fBhw9ZTBK9v

Reply:
Oh, my question seems to have been deleted. What would you recommend that I do with the error as createSOMASServer does not use it? Just panic?
I was hoping that this problem would have been picked up before this point if there weren't any clients added

Reply:
https://play.golang.org/p/AStpwETnESL <- this is cleaner :) uses more memory though

Reply:
Hope it's better now

Reply:
I'll add a few more tests